src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list-items">
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              data-qa="nav-hover"
+              href="#"
+              class="nav__list-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#"
+              class="nav__list-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product photo"
```

Comment:
The image path should be relative to the location of the HTML file, not the CSS or SCSS files. If the images directory is at the root of the project, the path should be 'images/imac.jpeg' instead of './images/imac.jpeg'.